javascripts/govuk/selection-buttons.js

```diff
@@ -1,3 +1,6 @@
+// DEPRECATED
+// This isn’t needed if you’re using GOV.UK Elements 3.0.0 or above
+
 ;(function (global) {
```

Comment:
Could we add a `console.warn('Deprecated this is going away blah blah pls do the cool new way')`?